channels-rke2.yaml

```diff
@@ -1048,4 +1076,25 @@ releases:
       harvester-csi-driver:
         repo: rancher-rke2-charts
         version: 0.1.1500
+      rke2-cilium:
+        repo: rancher-rke2-charts
+        version: 1.12.500
+      rke2-canal:
+        repo: rancher-rke2-charts
+        version: v3.25.0-build2023020901
+      rke2-calico:
+        repo: rancher-rke2-charts
+        version: v3.25.001
+      rke2-calico-crd:
+        repo: rancher-rke2-charts
+        version: v3.25.001
+      rke2-snapshot-controller:
+        repo: rancher-rke2-charts
+        version: 1.7.201
+      rke2-snapshot-controller-crd:
+        repo: rancher-rke2-charts
+        version: 1.7.201
+      rke2-snapshot-validation-webhook:
+        repo: rancher-rke2-charts
+        version: 1.7.100
```

Comment:
These are net-new charts for rke2. There isn't anything we need to do differently here, just calling it out in case it becomes relevant in the future.